### PR TITLE
Accessibility Suite Integration

### PR DESCRIPTION
**Pull request by astaiglesia**
# Accessibility Testing Suite Feature
This pull request is being created to merge a feature branch to provide accessibility testing functionality to the existing application. This initial merge will integrate a simple React component structure + Sass styling of a new testing suite leveraging the existing Context API + Hooks state management system.
The testing team is to subsequently bring test code generating functionality to the new suite, utilizing an Axe-core testing library within the Jest framework. Stretch features will add additional functionality and components to the accessibility suite.
<hr>

# Components included in this merge :
<br>

- A new button selection at the application's test suite entry point

Screen Shot 2021-03-11 at 1 57 46 PM

- A test menu to provide button functionality for New Test, Preview, Run Test, and Need Help options.

Screen Shot 2021-03-11 at 1 58 47 PM

- The minimum viable component for the accessibility testing describe block

Screen Shot 2021-03-11 at 1 58 53 PM

- The ability to populate additional describe blocks and it statements

Screen Shot 2021-03-11 at 2 00 27 PM

- A file selection field

Screen Shot 2021-03-11 at 1 58 34 PM

Iterations to the accessibility test suite can provide additional user configuration options for the describe block including the ability to configure the axe-core API call, as well as provide integration testing options.

Co-authored-by: Sharon Zhu | sharonzhu.15@gmail.com
Co-authored-by: Alfred Sta. Iglesia | a.sta.iglesia@gmail.com

### CODE REVIEW

**Review by annieshinn — APPROVED** [review]
Good evening team, 

I've pulled down your code and tested it in a tester branch on my local repository. As described, I see a new "Accessibility" button that leads to an accessibility testing interface. So many new components - AWESOME JOB!

I've made 4 comments throughout the code - **please note them as fixes/conversations for a future sprint**. I note some minor bugs in the app, but nothing important is breaking or broken, so awesome job team!

Also - I have a little inkling that the state is not appropriately being passed down/causing the new components to re-render with updated state. I'll demo how I found this out at our next full-group code review. Again, this can be **saved for a future sprint.**

I'm approving this PR, since what is working is what's been described! The nuances can be worked out as we continue on. 

Thanks team!
Annie S

File: package.json [hunk 1]
```diff
@@ -69,10 +69,11 @@
     "@testing-library/jest-dom": "^5.8.0",
     "@testing-library/react": "^8.0.9",
     "@testing-library/react-hooks": "^3.2.1",
+    "@types/classnames": "^2.2.11",
     "@types/jest": "^25.2.3",
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.33.0",
-    "electron": "^5.0.3",
+    "electron": "^12.0.1",
```
[the remark this answers is not in the document]
Comment (annieshinn) [reply package.json:76]: Resolved: We've agreed to remove this file's changes from the PR.

[the remark this answers is not in the document]
Comment (taoantaoan) [reply package.json]: As per discussed, let's try and revert back to the original package file!

**Review by taoantaoan — APPROVED** [review]
Awesome work.

**Review by annieshinn — APPROVED** [review]
Thanks for the updates. Approved.

File: src/components/TestCase/AccTestCase.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  return (
+    
+      <div id={styles.AccTestCase}>
+        <div id='head'>
+          <AccTestMenu />
+        </div>
+        <section id={styles.testCaseHeader}>
+        <label htmlFor='fileImport'>Import File From</label>
+          <div id={styles.labelInput} style={{ width: '80%' }}>
+            <SearchInput
```
[the remark this answers is not in the document]
Comment (astaiglesia) [reply src/components/TestCase/AccTestCase.jsx:56]: Thanks Annie! 
We will look into correcting this bug as part of a subsequent PR.

File: src/components/AccTestComponent/ItRenderer/ItRenderer.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  const deleteItStatementHandleClick = (e) => {
+    const itId = e.target.id;
+    dispatchToAccTestCase(deleteItStatement(describeId, itId));
+  };
+
+  return filteredIds.map((id, i) => (
+    <div id={styles.ItRenderer} key={i}>
+
+      <i
+        onClick={deleteItStatementHandleClick}
```
[the remark this answers is not in the document]
Comment (astaiglesia) [reply src/components/AccTestComponent/ItRenderer/ItRenderer.tsx:43]: Thanks Annie! 
We have fixed this issue and is now part of this current pull request.

File: src/components/TestMenu/AccTestMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+          />
+          {/* Just send user to docs on button click */}
+        </div>
+
+        <div
+          id={styles.right}
+          style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center' }}
+        >
+          <button data-testid='addDescribeButton' onClick={handleAddDescribeBlock}>
+            +Describe Block
```
[the remark this answers is not in the document]
Comment (astaiglesia) [reply src/components/TestMenu/AccTestMenu.tsx:80]: Thanks Annie! 
We will look into correcting this bug as part of a subsequent PR.

[the remark this answers is not in the document]
Comment (annieshinn) [reply src/components/TestMenu/AccTestMenu.tsx:80]: Fixed with #11

File: src/components/AccTestComponent/ItRenderer/ItRenderer.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (annieshinn) [reply src/components/AccTestComponent/ItRenderer/ItRenderer.tsx:43]: Fixed with #11

**Pull request merged**